ArduCopter/mode_flip.cpp

```diff
@@ -103,27 +103,27 @@ void ModeFlip::run()
     motors->set_desired_spool_state(AP_Motors::DesiredSpoolState::THROTTLE_UNLIMITED);
 
     // get corrected angle based on direction and axis of rotation
-    // we flip the sign of flip_angle_cd to minimize the code repetition
-    int32_t flip_angle_cd;
+    // we flip the sign of flip_angle to minimize the code repetition
+    float flip_angle_rad;
 
     if (roll_dir != 0) {
-        flip_angle_cd = ahrs.roll_sensor * roll_dir;
+        flip_angle_rad = ahrs.get_roll_rad() * roll_dir;
     } else {
-        flip_angle_cd = ahrs.pitch_sensor * pitch_dir;
+        flip_angle_rad = ahrs.get_pitch_rad() * pitch_dir;
     }
 
     // state machine
     switch (_state) {
 
     case FlipState::Start:
         // under 45 degrees request 400deg/sec roll or pitch
-        attitude_control->input_rate_bf_roll_pitch_yaw_cds(FLIP_ROTATION_RATE_CDS * roll_dir, FLIP_ROTATION_RATE_CDS * pitch_dir, 0.0);
+        attitude_control->input_rate_bf_roll_pitch_yaw_rads(cd_to_rad(FLIP_ROTATION_RATE_CDS) * roll_dir, cd_to_rad(FLIP_ROTATION_RATE_CDS) * pitch_dir, 0.0);
```

Comment:
we convert FLIP_ROTATION_RATE_CDS to radians so many times it might be best to just change the defintion